boards/arm/stm32f411e_disco/stm32f411e_disco.dts

```diff
@@ -79,3 +79,7 @@
 &rtc {
 	status = "okay";
 };
+
+&usbotg_fs {
```

Comment:
Is it necessary to name it `&usbotg_fs`? We have one build fail issue with USB Audio samples which were merged for 2.3 release. I do not know if `STM32F411E Disco` support iso endpoints but maybe it does. The issue is Audio uses DT for device configuration and there is one DT overlay provided for both samples ([app.overlay](https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/subsys/usb/audio/headset/app.overlay#L7)) which refers to `&usbd`. Tyring to build for ST boards fails because of that. Do You know any policy there or if it must be named like that?

Reply:
@emob-nordic I don't think your question is relevant to this PR as `&usbotg_fs` is defined under dts/arm/st... and already used on more than 20 ST boards.  Please raise a dedicated issue to discuss this point.

Reply:
But it can be simple fixed here, see my [comment](https://github.com/zephyrproject-rtos/zephyr/issues/25313#issuecomment-628488590)

Reply:
> @emob-nordic I don't think your question is relevant to this PR as `&usbotg_fs` is defined under dts/arm/st... and already used on more than 20 ST boards. Please raise a dedicated issue to discuss this point.

Yes, this is the simple answer to why it is named that way. I was using the other st boards as a reference to remain consistent. 

I am happy to change it to something else if needed, just let me know what you suggest.

Reply:
> But it can be fixed here, see my [comment](https://github.com/zephyrproject-rtos/zephyr/issues/25313#issuecomment-628488590)

I went with the recommendation you made in this comment. If this isn't what you were looking for, let me know.